src/features/Students/StudentsFilters/index.jsx

```diff
@@ -34,7 +34,7 @@ const initialState = {
 };
 
 const StudentsFilters = ({ resetPagination, fetchData, setFilters }) => {
```

Comment:
I suppose we will move this component to redux. Is that correct? @AuraAlba

Reply:
Yes, in the other ticket the student and courses pages will be moved to redux